packages/office-ui-fabric-react/src/components/pickers/BasePicker.test.tsx

```diff
@@ -319,13 +321,179 @@ describe('BasePicker', () => {
+
+    ReactDOM.render(
+      <BasePickerWithType
+        onResolveSuggestions={onResolveSuggestions}
+        onRenderItem={onRenderItem}
+        onRenderSuggestionsItem={basicSuggestionRenderer}
+        componentRef={picker}
+      />,
+      root,
+    );
+
+    const input = document.querySelector('.ms-BasePicker-input') as HTMLInputElement;
+    input.focus();
+    input.value = 'asdff';
+    ReactTestUtils.Simulate.input(input);
+    jest.runAllTimers();
+
+    expect(getSuggestions(document)).toBeTruthy();
+
+    ReactTestUtils.Simulate.keyDown(input, { which: 27 });
+
+    expect(getSuggestions(document)).toBeFalsy();
+    ReactTestUtils.Simulate.click(input, { button: 0 });
+
+    jest.runAllTimers();
+
+    expect(getSuggestions).toBeTruthy();
+  });
+
+  it('Opens menu on click when suggestions has been dismissed', () => {
+    jest.useFakeTimers();
+    document.body.appendChild(root);
+
+    const picker = React.createRef<IBasePicker<ISimple>>();
+
+    ReactDOM.render(
+      <BasePickerWithType
+        onResolveSuggestions={onResolveSuggestions}
+        onRenderItem={onRenderItem}
+        onRenderSuggestionsItem={basicSuggestionRenderer}
+        componentRef={picker}
+      />,
+      root,
+    );
+
+    const input = document.querySelector('.ms-BasePicker-input') as HTMLInputElement;
+    input.focus();
+    input.value = 'asdff';
+    ReactTestUtils.Simulate.input(input);
+    jest.runAllTimers();
+
+    expect(getSuggestions(document)).toBeTruthy();
+
+    ReactTestUtils.Simulate.keyDown(input, { which: 27 });
+
+    expect(getSuggestions(document)).toBeFalsy();
+    ReactTestUtils.Simulate.click(input, { button: 0 });
+
+    jest.runAllTimers();
+
+    expect(getSuggestions(document)).toBeTruthy();
+  });
+
+  it('Opens menu when input refocused after search has happened', () => {
+    jest.useFakeTimers();
+    document.body.appendChild(root);
+
+    const picker = React.createRef<IBasePicker<ISimple>>();
+
+    ReactDOM.render(
+      <div>
+        <BasePickerWithType
+          onResolveSuggestions={onResolveSuggestions}
+          onRenderItem={onRenderItem}
+          onRenderSuggestionsItem={basicSuggestionRenderer}
+          componentRef={picker}
+        />
+        <button id="toFocus">focus me</button>
+      </div>,
+      root,
+    );
+
+    const input = document.querySelector('.ms-BasePicker-input') as HTMLInputElement;
+    input.focus();
+    input.value = 'bl';
+    ReactTestUtils.Simulate.input(input);
+    jest.runAllTimers();
+
+    expect(getSuggestions(document)).toBeTruthy();
+
+    (document.querySelector('#toFocus') as any).focus();
+
+    // Implicit test to ensure suggestions are dismissed when focus lost
+    expect(getSuggestions(document)).toBeFalsy();
+
+    jest.runAllTimers();
+    input.focus();
+    jest.runAllTimers();
+
+    expect(getSuggestions(document)).toBeTruthy();
+  });
+
+  it('Opens calls onResolveSuggestions if it currently doesnt have suggestions', () => {
```

Comment:
nit:
```suggestion
  it('Opens calls onResolveSuggestions if it currently doesn't have suggestions', () => {
```

Reply:
I'm going to leave this as is for right now. It needs an escape string to make the apostrophe work.

Reply:
You could always use `does not` instead of `doesn't`.